components/TokenSeries/CardList.js

```diff
@@ -146,6 +148,21 @@ const TokenSeriesSingle = ({
 		},
 	})
 
+	const { data: ownedToken } = useSWRImmutable(
+		_token && currentUser
+			? {
+					contract_id: _token.contract_id,
+					token_series_id: _token.token_series_id,
+					owner_id: currentUser,
+			  }
+			: null,
+		(key) => {
+			return ParasRequest.get(`${process.env.V2_API_URL}/token`, {
+				params: key,
+			})
+		}
+	)
+
 	const store = useStore()
```

Comment:
ini emang supposed to fetch /token lagi di CardList? soalnya kalo gua liat network jadi bakal banyak fetch single token dari setiap item di /token-series. 

jadi misal fetch `/token-series` => Array(10), bakal fetch `/token` 10x

any suggestion ga? atau mungkin bisa di kasih conditional aja kalo dia category c4c aja fetchnya

Reply:
bisa sih hu dikasih conditional fetch cuma di category nya c4c